l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -276,17 +276,12 @@ def _prepare_generic_line_data(self, line):
                     ('type_tax_use', '=', 'purchase'),
                     ('kind_id.code', '=', line.Natura),
                     ('amount', '=', 0.0),
-                ])
+                ], order='sequence', limit=1)
```

Comment:
Piuttosto che prelevare la prima utile senza segnalare nulla a nessuno, io lascerei il log dell'inconstistenza e preleverei successivamente la prima aliquota utile. In questo modulo si tiene traccia dell'inconsistenza dei dati ma non si lascia il valore vuoto.

Reply:
Il fatto è che avere diverse imposte con aliquota 0 è normale, per gestire le varie possibili esenzioni. Quindi in questo caso il sistema continuerebbe a segnalare inconsistenze

Reply:
@eLBati si lo scopo sarebbe appunto quello, inserire comunque la prima imposta valida trovata per priorità di sequenza, e segnalare comunque all'utente che ci sono altre possibili imposte
(p.s. perchè non hanno previsto dei banali campi 'art.' 'comma' 'legge' ecc. per definire in maniera precise le imposte non l'ho capito)

Reply:
Ok quindi togliere il `limit` e lasciare `if len(account_taxes) > 1`. Ok per me